gdal/frmts/aaigrid/aaigriddataset.cpp

```diff
@@ -1037,11 +1037,11 @@ GDALDataset *AAIGDataset::CommonOpen( GDALOpenInfo *poOpenInfo,
         // Scan for dot in subsequent chunks of data.
         while( !VSIFEofL(poDS->fp) )
         {
-            CPL_IGNORE_RET_VAL(VSIFReadL(pabyChunk, nChunkSize, 1, poDS->fp));
+            const size_t nLen = VSIFReadL(pabyChunk, nChunkSize, 1, poDS->fp);
```

Comment:
Should likely be to fix issue raised by autotest:
```suggestion
            const size_t nLen = VSIFReadL(pabyChunk, 1, nChunkSize, poDS->fp);
```